processor/src/test/java/org/mapstruct/ap/test/inheritfromconfig/CarMapperWithAutoInheritance.java

```diff
@@ -26,24 +25,24 @@
 import org.mapstruct.factory.Mappers;
 
 /**
- * @author Sjaak Derksen
+ * @author Andreas Gudian
+ *
  */
-@Mapper
-public interface SourceTargetMapperErroneouslyAnnotated2 {
-
-    SourceTargetMapperErroneouslyAnnotated2 INSTANCE
-            = Mappers.getMapper( SourceTargetMapperErroneouslyAnnotated2.class );
+@Mapper(
+    config = AutoInheritedConfig.class
+)
+public interface CarMapperWithAutoInheritance {
```

Comment:
Is there a test for auto-inheriting from a method declared on the mapper itself rather than on a config class?